package.json

```diff
@@ -14,7 +14,8 @@
     "brave-core-ios": "https://github.com/brave/brave-browser/releases/download/v1.38.73/brave-core-ios-1.38.73.tgz",
     "page-metadata-parser": "^1.1.3",
     "readability": "github:mozilla/readability#b9f47bcc8d3c223cabe2dec6a42eeb3bd778d85c",
-    "webpack-cli": "^4.8.0"
+    "webpack-cli": "^4.8.0",
+    "events": "^3.3.0"
```

Comment:
Maybe worth mentioning in the security review that we add this dep?
from package-lock looks like we use it already though

Reply:
Oh, this can actually be removed! I forgot I added it prior to using 1.38.x and being able to grab the ethereum provider JS from brave-core

Reply:
@diracdeltas shouldn't we use precise versions in our `package.json` to avoid package takeover issues?

Reply:
The events package here is actually no longer included so there's no change here anymore.

We can always lock in precise versions of everything in our package.json though in a different PR if needed :)